pytorch_pfn_extras/onnx/strip_large_tensor.py

```diff
@@ -27,8 +27,8 @@ def _is_stripped(tensor):
             return external_value_dict.get('type', '') == 'stripped'
         except ValueError:
             # Invalid JSON, indicating `external_data.value` contains
-            # a file path
-            continue
+            # a file path, assume the tensor has no value means already stripped
```

Comment:
This fixes flake8 (line length limit) error. Correct me if my understanding is wrong 😃 
```suggestion
            # a file path. Treat the tensor as if it is already stripped.
```

Reply:
As this is comment-only fix, let me add this fix so that we can include this PR in the release today. Feel free to let me know if you have any concern 😄